bridge/certificatesSSM.ts

```diff
@@ -0,0 +1,83 @@
+import type { SSMClient } from '@aws-sdk/client-ssm'
+import { STACK_NAME } from '../cdk/stacks/stackConfig.js'
+import { getSettingsOptional, putSettings } from '../util/settings.js'
+import type { CAFiles } from './caLocation.js'
+import { readFile, writeFile } from 'node:fs/promises'
+import type { CertificateFiles } from './mqttBridgeCertificateLocation.js'
+import type { logFn } from '../cli/log.js'
+
+export const backupCertificatesToSSM = async ({
+	ssm,
+	parameterNamePrefix,
+	certificates,
+	debug,
+}: {
+	ssm: SSMClient
+	parameterNamePrefix: string
+	debug?: logFn
+	certificates: CAFiles | CertificateFiles
+}): Promise<void> => {
+	debug?.(`Writing to SSM`)
+	await Promise.all(
+		Object.entries(certificates).map(async ([k, v]) =>
+			putSettings({
+				ssm,
+				stackName: STACK_NAME,
+				scope: 'codebuild',
+				system: 'stack',
+			})({
+				property: `${parameterNamePrefix}_${k}`,
+				value: await readFile(v, { encoding: 'utf-8' }),
+			}),
+		),
+	)
+}
+
+export const restoreCertificatesFromSSM = async ({
+	ssm,
+	parameterNamePrefix,
+	certificates,
+	debug,
+}: {
+	ssm: SSMClient
+	parameterNamePrefix: string
+	certificates: CAFiles | CertificateFiles
+	debug?: logFn
+}): Promise<boolean> => {
+	let hasRestored = false
+
+	const parameters = await getSettingsOptional<Record<string, string>, null>({
+		ssm,
+		stackName: STACK_NAME,
+		scope: 'codebuild',
```

Comment:
We don't use [codebuild](https://docs.aws.amazon.com/codebuild/latest/userguide/welcome.html).